src/styles/blocks/content.scss

```diff
@@ -0,0 +1,9 @@
+.content {
+  display: flex;
+  justify-content: center;
+
+  &__catalog {
+    max-width: 1024px;
```

Comment:
You should calculate width based o nother variables.
What if the card width will change?